test/sql/util.sql

```diff
@@ -31,3 +31,18 @@ SELECT * FROM (
        (_timescaledb_functions.makeaclitem(:'TMP_USER', 0, 'insert,select', true))
     ) AS t(item);
 DROP USER :TMP_USER;
+
+SELECT _timescaledb_functions.align_to_bucket('5 minutes'::interval, rng::tstzrange)
+FROM (VALUES
+         ('["2025-04-25 11:10:00+02","2025-04-25 11:14:00+02"]'),
+         ('["2025-04-25 11:10:00+02","2025-04-25 11:17:00+02"]'),
+         ('["2025-04-25 11:10:00+02","2025-04-25 11:20:00+02")')
+     ) t(rng);
+
+SELECT typ,
+       _timescaledb_functions.get_min_for_type(typ),
+       _timescaledb_functions.get_max_for_type(typ)
```

Comment:
Same here. Tests for invalid, unexpected inputs.

Reply:
Added here as well.